troposphere/static/js/components/common/image_request/ImageLicensing.jsx

```diff
@@ -17,7 +17,7 @@ export default React.createClass({
         });
     },
     removeLicenseFromList: function(license_id) {
-        console.log("Remove", license_id);
+        // console.log("Remove", license_id);
```

Comment:
I think we might just want to `git rm troposphere/static/js/components/common/image_request/ImageLicensing.jsx`.